Extensions/xinbox.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE XInbox **//
-//* VERSION 1.9.1 **//
+//* VERSION 1.10 **//
```

Comment:
Yeah, please keep the x.y.z format even if z is 0.

On Mon, Aug 17, 2015, 19:39 Evan <notifications@github.com> wrote:

> In Extensions/xinbox.js
> <https://github.com/new-xkit/XKit/pull/304#discussion_r37214054>:
>
> > @@ -1,5 +1,5 @@
> >  //* TITLE XInbox **//
> > -//* VERSION 1.9.1 **//
> > +//* VERSION 1.10 **//
>
> maybe should be 10.0?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/new-xkit/XKit/pull/304/files#r37214054>.
>